### PR TITLE
Feature/rat texts

### PR DESCRIPTION
**Pull request by nickguendling**
## Description
RATs get some more special text treatment:
- Showing an additional hint on the success view controller that the user should verify the result with a PCR test
- Showing "Ihr Schnelltest wurde hinzugefügt" also on invalid and expired tests.

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift [hunk 1]
```diff
@@ -1051,7 +1051,13 @@ class ExposureSubmissionCoordinator: NSObject, RequiresAppDependencies {
 	}
 	
 	private func showExposureSubmissionSuccessViewController() {
+		guard let coronaTestType = model.coronaTestType else {
+			dismiss()
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift:1055]: perhaps add a log why we dismiss here?

**Comment by marcussc** [comment]
No Jira Ticket?

**Review by marcussc — APPROVED** [review]

**Comment by nickguendling** [comment]
> No Jira Ticket?
Nope, just one thing that UX found and two things that I found myself, didn't create a ticket for them.

**Comment by marcussc** [comment]
> No Jira Ticket?
> Nope, just one thing that UX found and two things that I found myself, didn't create a ticket for them.

Okay just watching out that this not getting out of control :)

**Review by janetback — APPROVED** [review]
UA reviewed.

**Pull request merged**